docs/autogen/templates/getting_started.rst.jinja

```diff
@@ -0,0 +1,726 @@
+install ``meson`` via ``pip3`` since the version in the package registry is too
+old for SPDK, thus if installed via the package manager then you will
+experience build errors as the xNVMe build system starts building SPDK.
+
+Once you have the full source of xNVMe, third-party library dependencies, and
+setup the toolchain then run the following to ensure that the xNVMe repository
+is clean from any artifacts left behind by previous build failures::
+
+  make clobber
+
+And then go back to the :ref:`sec-building` and follow the steps there.
+
+Known Build Issues
+------------------
+
+If the above did not sort out your build-issues, then you might be facing one
+of the following known build-issues. If these do not apply to you, then please
+post an issue on GitHUB describing your build environment and output from the
+failed build.
+
+When building **xNVMe** on **Alpine Linux** you might encounter some issues due
+to musl standard library not being entirely compatible with **GLIBC** /
+**BSD**.
+
+The SPDK backend does not build on due to re-definition of ``STAILQ_*``
+macros. As a work-around, then disable the SPDK backend::
+
+  meson setup builddir -Dwith-spdk=false
+
+The Linux backend support for ``io_uring`` fails on **Alpine Linux** due to a
+missing definition in **musl** leading to this error message::
+
+  include/liburing.h:195:17: error: unknown type name 'loff_t'; did you mean
+  'off_t'?
+
+As a work-around, then disable ``io_uring`` support::
+
+  meson setup builddir -Dwith-liburing=false
+
+See more details on changing the default build-configuration of **xNVMe** in
+the section :ref:`sec-building-config`.
+
+.. _sec-building-custom:
+
+Customizing the Build
+=====================
+
+.. _sec-building-custom-toolchain:
+
+Non-GCC Toolchain
+-----------------
+
+To use a compiler other than **gcc**, then:
+
+1) Set the ``CC`` and ``CXX`` environment variable for the ``./configure`` script
+2) Pass ``CC``, and ``CXX`` as arguments to ``make``
+
+For example, compiling **xNVMe** on a system where the default compiler is not
+**gcc**:
+
+.. code-block:: bash
+
+  CC=gcc CXX=g++ ./configure <YOUR_OPTIONS_HERE>
+  make CC=gcc CXX=g++
+  make install CC=gcc CXX=g++
+
+Recent versions of **icc**, **clang**, and **pgi** should at be able to satisfy
+the **C11** and **pthreads** requirements. However, it will most likely require
+a bit of fiddling.
+
+.. note:: The **icc** works well after you bought a license and installed it
+  correctly. There is a free option with Intel System Suite 2019.
+
+.. note:: The **pgi** compiler has some issues linking with **SPDK/DPDK** due
+  to unstable **ABI** for **RTE** it seems.
+
+The path of least fiddling around is to just install the toolchain and
+libraries as described in the :ref:`sec-building-toolchain` section.
+
+.. _sec-building-config:
+
+Custom Configuration
+--------------------
+
+See the list of options in ``meson_options.txt``, this file options the
```

Comment:
this file options

Maybe a different word than options should be used here?

Reply:
fixed.